test/operations/manager.go

```diff
@@ -488,19 +488,19 @@ func (m *Manager) CheckAccountTokenBalance(ctx context.Context, network NetworkS
 }
 
 // GetClaimData gets the claim data
-func (m *Manager) GetClaimData(ctx context.Context, networkID, depositCount uint) ([][bridgectrl.KeyLen]byte, *etherman.GlobalExitRoot, error) {
+func (m *Manager) GetClaimData(ctx context.Context, networkID, depositCount uint) ([bridgectrl.KeyLen][bridgectrl.KeyLen]byte, *etherman.GlobalExitRoot, error) {
```

Comment:
we are not able to `KeyLen` as the 1st dimension, it is same as with `MerkleTree.Height` (not 32)

Reply:
As this height can not be defined as variable, I have decided to define it as constant. If this is not good enough we could create an issue to not forget it.

Reply:
OK, fair enough, this is only limited to the test toolkit, I think we can ignore